driver-core/src/main/java/com/datastax/driver/core/Metadata.java

```diff
@@ -111,17 +111,19 @@ Host getHost(InetSocketAddress address) {
         return hosts.values();
     }
 
-    /*
+    /**
      * Deal with case sensitivity for a given element id (keyspace, table, column, etc.)
-     *
+     * <p/>
      * This method is used to convert identifiers provided by the client (through methods such as getKeyspace(String)),
      * to the format used internally by the driver.
-     *
+     * <p/>
      * We expect client-facing APIs to behave like cqlsh, that is:
-     * - identifiers that are mixed-case or contain special characters should be quoted.
-     * - unquoted identifiers will be lowercased: getKeyspace("Foo") will look for a keyspace named "foo"
+     * <ul>
+     * <li>identifiers that are mixed-case or contain special characters should be quoted.</li>
+     * <li>unquoted identifiers will be lowercased: getKeyspace("Foo") will look for a keyspace named "foo".</li>
+     * </ul>
      */
-    static String handleId(String id) {
+    public static String handleId(String id) {
```

Comment:
This was already duplicated in the query builder; now that it's used in 3 places, I think it makes sense to make it public.